examples/rust/robby_fischer/README.md

```diff
@@ -0,0 +1,317 @@
+<!--[metadata]
+title = "Robby fischer"
+tags = ["3D", "URDF", "Blueprint"]
+source = "https://github.com/02alexander/robby-fischer/tree/urdf-vis"
+thumbnail = "https://static.rerun.io/robby_thumbnail/71d2d57e9720e7a96e35a43467b5d2c45aa716d9/480w.png"
+thumbnail_dimensions = [480, 385]
+-->
+
+https://vimeo.com/989548054?autoplay=1&loop=1&autopause=0&background=1&muted=1&ratio=1920:1080
+
+## Used Rerun types
+
+[`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d), [`Points2D`](https://www.rerun.io/docs/reference/types/archetypes/points2d), [`Boxes3D`](https://www.rerun.io/docs/reference/types/archetypes/boxes3d), [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole), [`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Mesh3D`](https://www.rerun.io/docs/reference/types/archetypes/mesh3d), [`LineStrips3D`](https://www.rerun.io/docs/reference/types/archetypes/line_strips3d),
+
+## Background
+Robby Fischer is an autonomous robot arm that you can play chess against, created by Alexander Berntsson and Herman Lauenstein. It detects the human's move by using a camera that watches which squares has a piece on it and what color that piece is. It doesn't need to see recognize different roles (pawn, rook, etc..) visually because it knows the start position so it can always figure out what piece stand on what square, although it's a bit problematic if the human promotes a pawn because robot must figure out which piece the pawn promoted to. This is why it also looks at the adjacent white board, it has a specific location associated with each piece, so if the human promotes to a queen the queen square will be empty and Robby can figure out that the human promoted to a queen.
```

Comment:
```suggestion
Robby Fischer is an autonomous robot arm that you can play chess against, created by Alexander Berntsson and Herman Lauenstein. It detects the human's move by using a camera that watches which squares has a piece on it and what color that piece is. It doesn't need to see recognize different roles (pawn, rook, etc..) visually because it knows the start position so it can always figure out what piece stand on what square. However, this is a bit problematic if the human promotes a pawn because robot must figure out which piece the pawn was promoted to. This is why it also looks at the adjacent white board, where it has a specific location associated with each piece, so if the human promotes to a queen the queen square will be empty and Robby can figure out that the human promoted to a queen.
```
Unclear to me what is meant by the 'adjacent white board' here - is it where the removed pieces are supposed to go? Don't quite follow this part

Reply:
that's mentioned indeed later. Worth clarifying!